pennylane/_qubit_device.py

```diff
@@ -1273,6 +1286,65 @@ def classical_shadow(self, wires, circuit, seed=None):
 
         return self._cast(self._stack([outcomes, recipes]), dtype=np.int8)
 
+    def shadow_expval(self, obs, circuit):
+        r"""Compute expectation values using classical shadows in a differentiable manner.
+
+        The canonical way of computing expectation values is to simply average the expectation values for each local snapshot, :math:`\langle O \rangle = \sum_t \text{tr}(\rho^{(t)}O) / T`.
+        This corresponds to the case ``k=1``. However, it is often desirable for better accuracy to split the ``T`` measurements into ``k`` equal parts to compute the median of means, see `2002.08953 <https://arxiv.org/abs/2002.08953>`_.
+
+        One of the main perks of classical shadows is being able to compute many different expectation values by classically post-processing the same measurements. This is helpful in general as it may help
+        save quantum circuit executions.
```

Comment:
Nice! Could be cool to use `qml.tracker` to show the number of executions happening :)

Reply:
Actually as per last Friday's realizations I think that it is actually not true 😢 Comes down to measuring qubit-wise commuting observables (+ some additional unnecessary measurements that are being discarded).

Some of the things in the doc-strings will need to be rephrased to not further feed into this widespread misconception!

Reply:
I updated the doc-string to refrain from such strong statements and made a note that in most situations one is better advised to perform qubit-wise commuting groups simultaneously.